composables/braintree.ts

```diff
@@ -8,10 +8,12 @@ import {
 import { type PayEvent } from '~/types/common';
 
 export const useBraintree = () => {
+  const { t } = useI18n();
```

Comment:
does this work now? before I would get errors when using it in composables.

Reply:
yes it works...